pandas/core/frame.py

```diff
@@ -4363,8 +4363,9 @@ def quantile(self, q=0.5, axis=0, numeric_only=True):
         ----------
         q : float or array-like, default 0.5 (50% quantile)
             0 <= q <= 1, the quantile(s) to compute
-        axis : {0, 1}
-            0 for row-wise, 1 for column-wise
+        axis : {0, 1, 'index', 'columns'} (default 0)
+            0 or 'index' a quantile for each column, 1 or 'columns' a 
```

Comment:
I don't think this is as clear as the 0 for row-wise, ....  What about `0 or 'index' for row-wise, ...`?